api/src/main/java/ca/bc/gov/educ/api/batchgraduation/config/RestWebClient.java

```diff
@@ -27,7 +31,9 @@ public RestWebClient() {
 
     @Bean
     public WebClient webClient() {
-        return WebClient.builder().exchangeStrategies(ExchangeStrategies.builder()
+        return WebClient.builder()
+                .filter(setRequestHeaders())
```

Comment:
Please add a log() function as per other apis. You can see how others use it here: https://github.com/bcgov/EDUC-GRAD-GRADUATION-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/graduation/config/RestWebClient.java#L77